src/main/java/it/reply/orchestrator/dto/request/DeploymentRequest.java

```diff
@@ -54,9 +54,14 @@
   private String callback;
 
   @Nullable
-  @Min(value = 1, message = "Timeout value, if provided, must be at least of 1 minute")
+  @Min(value = 1, message = "Overall timeout value, if provided, must be at least of 1 minute")
   private Integer timeoutMins;
 
+  @Nullable
+  @Min(value = 1, message = "Provider timeout value, if provided, must be at least of 1 minute and"
+      + " equal or less than timeoutMins")
```

Comment:
`equal or less than timeoutMins` Do you actually do this validation?

Reply:
no, it is a circular reference (the order in which to assign the two variables is not fixed); it's just a message to the user

Reply:
>it is a circular reference (the order in which to assign the two variables is not fixed)

it's not circular; at the time the message comes to the REST controller both the field will or will not be populated. By the time you read the mesage, you can read both fields and validate them if you want

Reply:
ok i'll fix